gcp/config.ts

```diff
@@ -0,0 +1,18 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as utils from "./utils";
+
+let config = new pulumi.Config();
+
+export const GcpCloud = "gcp";
+export const LightbendNamespace = "lightbend";
+
+export const cloud = config.get<string>("cloud") || GcpCloud;
+export const zone = config.get<string>("gcp:zone") || "europe-west1-b";
+export const region = config.get<string>("gcp:region") || "europe-west1";
+export const operatorNamespace = config.get<string>("operator-namespace") || LightbendNamespace;
+export const operatorVersion = config.get<string>("operator-version") || "1.1.19";
```

Comment:
1.1.20 is out. Looks like the kind of thing that will very easily gets forgotten and out of date. Move to docs, no default, and make sure the docs has a variable for it?

Reply:
What do we do in code ? Should we fail fast if these are not set ? Like `zone` (see above) ?

Reply:
fail fast  👍